build.gradle

```diff
@@ -30,7 +30,7 @@ compileJava {
 }
 
 dependencies {
-    compile 'com.hazelcast.jet:hazelcast-jet:0.7-SNAPSHOT'
+    compile 'com.hazelcast.jet:hazelcast-jet:0.8-SNAPSHOT'
```

Comment:
0.8?

Reply:
leftover, it was to fail the build and see which resources are tried